quickevent/app/quickevent/plugins/CardReader/src/services/racomclient.cpp

```diff
@@ -132,6 +135,85 @@ void RacomClientSirxdConnection::onReadyRead()
 	}
 }
 
+RacomReadSplitFile::RacomReadSplitFile(QString fileName, int interval, int finishCode, QObject *parent)
+	: Super(parent)
+	, m_fileName(fileName)
+	, m_interval(interval)
+	, m_finishCode(finishCode)
+{
+	m_ReadTimer = new QTimer(this);
+	connect(m_ReadTimer, &QTimer::timeout, this, &RacomReadSplitFile::onTimerTimeOut);
```

Comment:
why not `connect(m_ReadTimer, &QTimer::timeout, this, &RacomReadSplitFile::redAndProcessFile);`

Reply:
I'll fix it